GOVERNANCE.md

```diff
@@ -679,24 +679,24 @@ Role:
 
 #### Accounts and services ownership and administration
 The PyMC Project also has accounts and hosts services on several platforms
-such as GitHub, Discourse, Twitter, ReadTheDocs or Medium.
+such as GitHub, Discourse, Twitter, ReadTheDocs, and Medium.
 
 If possible, all Council Members and relevant Core Contributors should have
-admin rights on those platforms.
-If this were not possible, admin rights should be distributed between
+administrative rights on those platforms.
+If this is not possible, administrative rights should be distributed among
 Council Members and relevant Core Contributors and establish a rotation
-of the admin rights every 1-2 years.
+of the administrative rights every 1-2 years.
 
 #### Permission examples
 This section lists some imaginary contributors with their teams and roles to
 provide examples on how to assign permissions:
 
 <details><summary>See permission examples</summary>
 
-- Arnau, recurrent contributor, discourse team
-  * Added to the Discourse PyMC_team and given "leader" trust level
+- Arnau, recurrent contributor, community team
+  * Added to the Community PyMC_team and given "leader" trust level
```

Comment:
This was a reference to the first point in discourse roles about adding members to the `PyMC_team` (or devs or something else, again, don't really know what I'm talking about) on Discourse. The roles look wrong on that end which is probably why this doesn't make much sense either.

Reply:
That is the idea, but we need to define something here that we agree on and makes sense. I have no idea how discourse teams work and wrote the current proposal :sweat_smile:. I wasn't sure it made much sense when I wrote it and I take @cluhmann struggling to understand what it meant as a confirmation that the doc needs updating and clarifying before we can go back on Discourse and make sure we follow what is defined here

Reply:
IMO being a contributor (of any type) and getting the premissions associated with that type are one and the same, a necessary and sufficient condition. If we write that core contributors of the dev team get write permissions to pymc repo, there should be no core dev contributors without nor people with write permissions who are not core dev contributors (or other role that gives those permissions like could be council member). 

I don't really know what "leader" trust level entails, but as a general rule, recurrent contributors should not have administrative permissions, that should be restricted to core contributors. If "leader" trust level is somewhat similar to triage permissions on github I'd vote for that.

Reply:
I would agree.  Just trying to highlight some of what (I think) we're looking for input/consensus on.